modules/phase_field/src/materials/DerivativeParsedMaterialHelper.C

```diff
@@ -21,6 +23,8 @@ DerivativeParsedMaterialHelper::DerivativeParsedMaterialHelper(const InputParame
                                                                VariableNameMappingMode map_mode) :
     ParsedMaterialHelper(parameters, map_mode),
     //_derivative_order(getParam<unsigned int>("derivative_order"))
```

Comment:
Actually this line will supersede the one below. ```third_derivatives``` already is a deprecated parameter. I plan to remove it entirely.